src/ConventionalCommitUtilities.js

```diff
@@ -2,10 +2,24 @@ import dedent from "dedent";
 import log from "npmlog";
 import path from "path";
 import semver from "semver";
+import conventionalChangelogPresetLoader from "conventional-changelog-preset-loader";
+import resolveFrom from "resolve-from";
 
 import ChildProcessUtilities from "./ChildProcessUtilities";
 import FileSystemUtilities from "./FileSystemUtilities";
 
+const allowedPresets = [
```

Comment:
Not a huge fan of copying in these names, but we might as well make it a `Set` so the test in `changelogConfigPath` can avoid the array `.includes` with `allowedPresets.has(presetName)`